components/canvas/Canvas.module.css

```
@@ -0,0 +1,13 @@
+.containermap {
```

Use camelcase to make classes better readable
```suggestion
.containerMap {
```